docs/user/00-10-overview-istio-controller.md

```diff
@@ -92,4 +92,8 @@ Run this command to get all resources created by the Istio module:
 
 ```bash
 kubectl get all|<resources-kind> -A -l kyma-project.io/module=istio
-```
\ No newline at end of file
+```
+
+## Compatibility Mode
+
+To enable compatibility mode in the Istio module, you can set the **spec.compatibilityMode** field in the Istio CR. This allows you to mitigate breaking changes when a new release introduces an Istio upgrade. The Istio module supports compatibility with the previous minor version of Istio. For example, the Istio module with Istio 1.21.0 applies a compatibility version of Istio 1.20. For more information, see [Compatibility Versions](https://istio.io/latest/docs/setup/additional-setup/compatibility-versions/).
```

Comment:
would you see them listed per Istio version or per Istio Module version?

Reply:
I think we should just have the current flags mentioned without a reference to the module version, because we will probably have them mentioned as well in the release notes or at least in the What's New, so users can decide rather they want the compatibility mode or not.